editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -352,7 +352,7 @@ export function renderCoreElement(
       const actualElement = conditionValue ? element.whenTrue : element.whenFalse
 
       if (elementPath != null) {
-        addFakeSpyEntry(metadataContext, elementPath, element, filePath, imports)
+        addFakeSpyEntry(metadataContext, elementPath, element, filePath, imports, conditionValue)
```

Comment:
Hmmm, I've removed this function from my branch completely...

Reply:
is it not used anymore or it just moved?

Reply:
so for the conditional itself, we should add the metadata entry. for the non-active-branch or contents of the non-active branch, we should _not_ have ElementInstanceMetadata

Reply:
Righto!